packages/hash/external-services/kratos/hooks/after.signup.onboard.jsonnet

```diff
@@ -0,0 +1,4 @@
+function(ctx) {
```

Comment:
I haven't worked with `.jsonnet` files before, but my understanding is they are used by kratos to determine what data to pass to a web-hook call. So for the after registration hook call, we provide the HASH workspace with the entire `identity` object for now.